src/sources/timezone_offset.ts

```diff
@@ -1,3 +1,14 @@
+import { toFloat } from '../utils/data'
+
 export default function getTimezoneOffset(): number {
```

Comment:
Does `timezoneOffset` even give extra entropy compared to the `timezone` component (the timezone name)? I.e. can there be fingerprints with the same `timezone` and different `timezoneOffset`? If no, then the `timezoneOffset` component can be removed completely.

Reply:
Yes, there are different timezones that can coincidentally have the same offset (e.g. Moscow and Dubai)

Reply:
My question is opposite. Can a single time zone JS value (e.g. `Europe/Moscow`) have different time offsets? It could due to the DST shift, but I've fixed it.

Reply:
I don't think it's possible, otherwise that would be multiple timezones.

Reply:
Ok, thanks. So the only reason to keep this component at the moment is to get entropy in IE.